src/snowflake/snowpark/udtf.py

```diff
@@ -421,6 +421,99 @@ class UDTFRegistration:
             -----------------------------
             <BLANKLINE>
 
+    [Preview Feature] Syntax for declaring vectorized UDTF process method is similar to above. Defining
```

Comment:
[Preview Feature] Syntax for declaring vectorized UDTF process method is similar to above.

->

[Preview Feature] The syntax for declaring a vectorized UDTF process method is similar to the above.

Reply:
Nit: We've been using the term "UDTF with a vectorized process/end_partition" instead of "vectorized UDTF process method", can you update the docs to match that so it's consistent? 
https://docs.snowflake.com/en/developer-guide/udf/python/udf-python-tabular-vectorized#getting-started